google/cloud/spanner_v1/pool.py

```diff
@@ -243,6 +243,7 @@ def bind(self, database):
             "CloudSpanner.FixedPool.BatchCreateSessions",
             observability_options=observability_options,
         ) as span:
+            attempt = 1
```

Comment:
nit: I don't think we need this. We can just pass in 1 as a constant in the function below just as for the other functions.